assets/containerisableComponents/contributionSelection/__tests__/contributionSelectionReducerTest.js

```diff
@@ -11,7 +11,7 @@ import { contributionSelectionActionsFor } from '../contributionSelectionActions
 describe('Contributions Selection reducer', () => {
 
   const PREFIX = 'MY_PREFIX';
-  const reducer = contributionSelectionReducerFor(PREFIX);
+  const reducer = contributionSelectionReducerFor(PREFIX, 'GBPCountries');
```

Comment:
Maybe we should test for all countries, or at least where there are differences?

Reply:
Added a new snapshot test for US.